Example/AztecUITests/AztecUITests.swift

```diff
@@ -0,0 +1,263 @@
+import XCTest
+
+class AztecSimpleTextFormattingTests: XCTestCase {
+
+    private var app: XCUIApplication!
+    private var htmlContentTextView: XCUIElement!
+    private var richContentTextView: XCUIElement!
+
+    override func setUp() {
+        super.setUp()
+
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+        XCUIDevice.shared().orientation = .portrait
+        app = XCUIApplication()
+        app.launch()
+
+        let tablesQuery = app.tables
+        tablesQuery.staticTexts[elementStringIDs.emptyDemo].tap()
+    }
+
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+
+    func testSimpleBoldText() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
+        richContentTextView.tap()
+        richContentTextView.enterAndSelectText(text: "text")
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.boldButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<strong>text</strong>"
+        XCTAssertEqual(expected, text)
+    }
+
+    func testSimpleItalicText() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
```

Comment:
this shows up in multiple places: I would add this to a helper eg enterTextInField().